Resources/Prototypes/SimpleStation14/Catalog/uplink_catalog.yml

```
@@ -138,3 +140,108 @@
     Bluespace: 1
   categories:
   - WizardPointless
+
+# Syndicate Uplink
+
+# let me know when money actually becomes useful to normal people
+# - type: listing
+#   id: Uplink
+#   name: Briefcase of cash
+#   description: contains 2 bundles of 10000 zorkmids. Good for bribery or purchasing goods for nefarious purposes.
+#   productEntity: BriefcaseBrownCash
+#   cost:
+#     Telecrystal: 7
+#   categories:
+#   - UplinkJob
+#   conditions:
+#   - !type:BuyerJobCondition
+#     whitelist:
+#     - Quartermaster
+
+- type: listing
+  id: UplinkBriefcaseMG
```

```suggestion
  id: UplinkBriefcaseSMG
```
No need for IDs to be obfuscated. Make sure you repeat this anywhere else it's used.